apps/backend/src/api/routes/v1/auth_register.py

```
@@ -0,0 +1,170 @@
+        raise
+    except Exception as e:
+        logger.error(f"Email verification error: {e}")
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="An error occurred during email verification"
+        )
+
+
+@router.post(
+    "/resend-verification",
+    response_model=Union[MessageResponse, ErrorResponse],
```

Ruff (0.11.9)

Use `X | Y` for type annotations

(UP007)